src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs

```diff
@@ -101,120 +101,122 @@ public override MetadataFile FindMetadataFile(Movie movie, string path)
+                        var streamDetails = new XElement("streamdetails");
+
+                        var video = new XElement("video");
+                        video.Add(new XElement("aspect", (float)movieFile.MediaInfo.Width / (float)movieFile.MediaInfo.Height));
+                        video.Add(new XElement("bitrate", movieFile.MediaInfo.VideoBitrate));
+                        video.Add(new XElement("codec", movieFile.MediaInfo.VideoCodec));
+                        video.Add(new XElement("framerate", movieFile.MediaInfo.VideoFps));
+                        video.Add(new XElement("height", movieFile.MediaInfo.Height));
+                        video.Add(new XElement("scantype", movieFile.MediaInfo.ScanType));
+                        video.Add(new XElement("width", movieFile.MediaInfo.Width));
+
+                        if (movieFile.MediaInfo.RunTime != null)
+                        {
+                            video.Add(new XElement("duration", movieFile.MediaInfo.RunTime.TotalMinutes));
+                            video.Add(new XElement("durationinseconds", movieFile.MediaInfo.RunTime.TotalSeconds));
+                        }
+
+                        streamDetails.Add(video);
+
+                        var audio = new XElement("audio");
+                        audio.Add(new XElement("bitrate", movieFile.MediaInfo.AudioBitrate));
+                        audio.Add(new XElement("channels", movieFile.MediaInfo.AudioChannels));
+                        audio.Add(new XElement("codec", GetAudioCodec(movieFile.MediaInfo.AudioFormat)));
+                        audio.Add(new XElement("language", movieFile.MediaInfo.AudioLanguages));
+                        streamDetails.Add(audio);
+
+                        if (movieFile.MediaInfo.Subtitles != null && movieFile.MediaInfo.Subtitles.Length > 0)
+                        {
+                            var subtitle = new XElement("subtitle");
+                            subtitle.Add(new XElement("language", movieFile.MediaInfo.Subtitles));
+                            streamDetails.Add(subtitle);
+                        }
+
+                        fileInfo.Add(streamDetails);
+                        details.Add(fileInfo);
                     }
 
-                    fileInfo.Add(streamDetails);
-                    details.Add(fileInfo);
-                }
+                    doc.Add(details);
+                    doc.Save(xw);
+
+                    xmlResult += doc.ToString();
+                    xmlResult += Environment.NewLine;
 
-                doc.Add(details);
-                doc.Save(xw);
+                }
+            }
+            if (Settings.MovieMetadataURL)
+            {
+                _logger.Debug("Generating Movie Metadata URL for: {0}", Path.Combine(movie.Path, movieFile.RelativePath));
 
-                xmlResult += doc.ToString();
+                xmlResult += "https://www.themoviedb.org/movie/" + movie.TmdbId;
```

Comment:
Are you sure you can just add the urls in the xml like this? Doesn't it need to be in a special element?

Reply:
It seems counter intuitive, but from what I saw in the doc, yes (point 2.3): https://kodi.wiki/view/NFO_files#Combination_nfo

Plex and other tools can also rely on this weird syntax